librad/src/uri.rs

```diff
@@ -316,7 +316,7 @@ impl RadUrn {
 
     pub fn as_rad_url_ref<'a>(&'a self, authority: &'a PeerId) -> RadUrlRef<'a> {
         RadUrlRef {
-            authority,
+            authority: &authority,
```

Comment:
Wait what? Why do we need to borrow here?

Reply:
Whoops, we don't. My guess is the compilers magic with referencing and dereferencing handled it for us.